change_log/next/CARBON-1818.yml

```diff
@@ -0,0 +1 @@
+Other: "[CARBON-1818](https://github.com/Sage/carbon/issues/1818) - Updated Alert to not extend Dialog, but render a Dialog instead."
```

Comment:
Jira link will be broken in the release notes. This will make things difficult for all the developers outside of Sage who use Carbon